src/main/java/org/opentripplanner/transit/raptor/rangeraptor/RangeRaptorWorker.java

```diff
@@ -241,17 +242,20 @@ private void findTransitForRound() {
             transitWorker.forEachBoarding(
               stopIndex,
               (int prevArrivalTime) -> {
-                boolean handled = boardWithConstrainedTransfer(
-                  txSearch,
-                  route.timetable(),
-                  stopIndex,
-                  stopPos,
-                  prevArrivalTime,
-                  boardSlack
-                );
+                boolean boardedUsingConstrainedTransfer =
+                  enableTransferConstraints &&
+                  txSearch != null &&
```

Comment:
```suggestion
```

Reply:
Sorry, missed this.

Reply:
The idea of the commit, was that txSearch can be null, if there are no constrained transfers for that pattern. If we remove this, we will get a NPE in `boardWithConstrainedTransfer`

Reply:
Ok, then its OK, i didn´t se that the txSearch could be `null`. I still want the `enableTransferConstraints` flag to be there. So then this is ok.

Reply:
We looked at this and @hannesj found a nice paper about method-dispatch, interesting reading: https://shipilev.net/blog/2015/black-magic-method-dispatch

We did decide on a NOOP implementation after @hannesj did some performance testing of the different approaches. He did not measure significant differences.